x/ccv/provider/keeper/proposal.go

```diff
@@ -309,9 +309,20 @@ func (k Keeper) ConsumerAdditionPropsToExecute(ctx sdk.Context) []types.Consumer
 	iterator := k.PendingConsumerAdditionPropIterator(ctx)
 	defer iterator.Close()
```

Comment:
`iterator` is never used in this function.

Reply:
Good catch!